src/Domain.LinnApps/PurchaseOrders/PurchaseOrder.cs

```diff
@@ -24,5 +24,7 @@ public class PurchaseOrder
         public string Overbook { get; set; }
 
         public decimal? OverbookQty { get; set; }
+
+        public string CurrencyCode { get; set; }
```

Comment:
You could make this a Currency as opposed to just a string since it'll probably need to be that eventually anyway